### PR TITLE
added ascii banner

### PR DESCRIPTION
**Pull request by RizkyRajitha**
added ASCII banner , an added a confirmation message when a new command is added.

### CODE REVIEW

**Review by shalithasuranga — CHANGES_REQUESTED** [review]
@RizkyRajitha  Nice .. Could you please use single quote instead of double

**Comment by RizkyRajitha** [comment]
can you specify where

**Comment by RizkyRajitha** [comment]
```
console.log(
    chalk.blue(
      figlet.textSync('RUNN', {
        font: 'Cyberlarge'
      })
    )
  );
```


like this

**Comment by shalithasuranga** [comment]
In all places actually. I  think we used single quotes initially. in yours changes I can see all are changed to double quotes. Could you match our existing coding style ?

**Comment by RizkyRajitha** [comment]
changed to single quotes

File: bin/runn.js [hunk 1]
```diff
@@ -1,19 +1,29 @@
...
+
+
+if (process.argv.length < 3) {
+  console.log(
+    chalk.blue(
+      figlet.textSync('RUNN', {
+        font: 'Cyberlarge'
+      })
+    )
+  );
```
Comment (shalithasuranga) [inline bin/runn.js:27]: Just noticed program.help(); is missing.  We need to show figlet + help when someone just enter the command (without parameters)

Comment (RizkyRajitha) [reply bin/runn.js:27]: snip

this is how currently working. 
how should i change this to

Comment (RizkyRajitha) [reply bin/runn.js:27]: ss

Comment (shalithasuranga) [reply bin/runn.js:27]: Hey.. Simply the outputs of two case of first image need to be same

Comment (RizkyRajitha) [reply bin/runn.js:27]: ahhh i got it , working on it .

**Review by shalithasuranga — APPROVED** [review]
👍  Nice work

**Comment by RizkyRajitha** [comment]
happy to contribute again 🙌

**Pull request merged**